cmake/Modules/Findopm-output.cmake

```diff
@@ -42,9 +42,10 @@ int main (void) {
 include (UseDynamicBoost)
 #debug_find_vars ("opm-output")
 
+
 if(OPM_OUTPUT_FOUND)
   get_filename_component(opm-output_PREFIX_DIR ${opm-output_LIBRARY} DIRECTORY)
-  find_program(OPM_OUTPUT_BINARIES test_compareSummary
+  find_program(OPM_OUTPUT_BINARIES test_RFT
```

Comment:
neither is correct. we want to find the 'compareSummary' binary, not the test. test_RFT was just a placeholder..